jina/clients/base/grpc.py

```diff
@@ -122,24 +129,36 @@ async def _unary_rpc(
                 continue_on_error=continue_on_error,
                 logger=self.logger,
             )
+            return resp
+
+        streamer = RequestStreamer(
+            request_handler=_request_handler,
+            result_handler=_result_handler,
+            iterate_sync_in_thread=False,
+            logger=self.logger,
+            **vars(self.args),
+        )
+        async for response in streamer.stream(
+                request_iterator=req_iter, results_in_order=results_in_order
+        ):
             if self.show_progress:
                 p_bar.update()
-            yield resp
+            yield response
 
     async def _get_results(
-        self,
-        inputs: 'InputType',
-        on_done: 'CallbackFnType',
-        on_error: Optional['CallbackFnType'] = None,
-        on_always: Optional['CallbackFnType'] = None,
-        compression: Optional[str] = None,
-        max_attempts: int = 1,
-        initial_backoff: float = 0.5,
-        max_backoff: float = 0.1,
-        backoff_multiplier: float = 1.5,
-        results_in_order: bool = False,
-        stream: bool = False,
-        **kwargs,
+            self,
+            inputs: 'InputType',
+            on_done: 'CallbackFnType',
+            on_error: Optional['CallbackFnType'] = None,
+            on_always: Optional['CallbackFnType'] = None,
+            compression: Optional[str] = None,
+            max_attempts: int = 1,
+            initial_backoff: float = 0.5,
+            max_backoff: float = 0.1,
+            backoff_multiplier: float = 1.5,
+            results_in_order: bool = False,
+            stream: bool = False,
```

Comment:
Small issue that we can fix later. This default value is different than the default value in the `.post` method. I forgot to update the default here last time.